src/main/java/org/glassfish/copyright/AbstractCopyright.java

```diff
@@ -332,7 +345,24 @@ else if (lc == null)
 	    System.out.println("No errors: " + file);
     }
 
-    /**
+	/**
+	 * Verifies if he file contains the message:
```

Comment:
nit typo:  "if the file.."

Reply:
https://github.com/eclipse-ee4j/glassfish-copyright-plugin/commit/620fdb207832e090dba2f0ac01aa3db89af8554e

Reply:
fixed in 620fdb2